### PR TITLE
Added licenses to master

### PR DESCRIPTION
**Pull request by ineiti**
Changed the license to MPLv2

### CODE REVIEW

**Comment by Daeinar** [comment]
Looks good.

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,375 @@
+This code is (c) by DEDIS/EPFL 2017 under the MPL v2 or later version.
```
Comment (nikkolasg) [inline LICENSE]: later versions ? are we sure ?

**Comment by nikkolasg** [comment]
Just a small question otherwise it's LGTM

File: LICENSE (hunk 1 above)
Comment (ineiti) [inline LICENSE]: That's commonly used, e.g. the GPL is mostly used as "or any later version" - supposing mozilla will not screw up.

**Pull request merged**